ci/setup_cmake.sh

```diff
@@ -30,17 +25,6 @@ else
   GOOGLETEST_FOLDER_PATH="googletest-${GOOGLETEST_VERSION}"
 fi
 
-cmake_install()
-{
-  tmp_dir=$(mktemp -d)
-  pushd $tmp_dir
-  wget https://github.com/Kitware/CMake/releases/download/v${CMAKE_VERSION}/cmake-${CMAKE_VERSION}-Linux-x86_64.sh
-  chmod +x cmake-${CMAKE_VERSION}-Linux-x86_64.sh
-  ./cmake-${CMAKE_VERSION}-Linux-x86_64.sh --prefix=/usr/local --skip-license
-  rm cmake-${CMAKE_VERSION}-Linux-x86_64.sh
-  popd
-}
-
```

Comment:
setup_cmake.sh script installing only the googletest - this seems confusing.
For the sake of consistency with naming, should we also add `apt-get install cmake` in this script ?

Reply:
This script is under `ci/`.

There is already another script, under `tools/setup-cmake.sh`, to also install cmake.

If the concern is that someone taking a dependency in `ci/*` will be affected, I can add a note in the changelog.

Reply:
Added changelog, and renamed to setup_googletest.sh